### DISCUSSION TITLE
🏆  PyTgCalls Contest!

### DISCUSSION THREAD

**Discussion by Laky-64** [pytgcalls/pytgcalls#199] [Announcements]
<img src="https://pytgcalls.github.io/src/assets/updates/PyTgCon24.png" alt="pytgcalls logo" />

# 🏆 #PyTgCon2K25
If you've crafted a public open repository, it's time to showcase your work in the PyTgCalls Contest! 🌟 

## ⁉️ Here's how to join:
- Visit the discussion page: PyTgCalls Contest (https://github.com/pytgcalls/pytgcalls/discussions/199)
- Share these details: Your Repository Name, Brief Description of Your Bot, Link to Your Repository

## 📈 UpVotes will be your ticket to recognition! 
The most upvoted bots will secure a prime spot on the PyTgCalls documentation website.

## 🥇 Prizes and Recognition
- **UpVotes as Tickets:** Every upvote is your ticket to recognition!
- **Prime Spot on Documentation Website:** The most upvoted bots will secure a prime spot on the PyTgCalls documentation website.

## 👮‍♀️ Contest Rules
- Make sure your repository goes beyond minor tweaks of a fork.
- Use the latest PyTgCalls version ([2.1.1](https://pypi.org/project/py-tgcalls/2.1.1/)).
- Your repository should be public.

_Join the contest now and showcase your PyTgCalls-powered bot for a chance to be featured prominently on the PyTgCalls documentation! Good luck!_

**Comment by AshokShau** [comment]
# 🏆 PyTgCon2K ~24~ - 25 – TgMusicBot Entry

Hey PyTgCalls team! 🚀 I’d love to submit **TgMusicBot** for the **PyTgCalls Contest**.  

🔹**Repository Name**: [TgMusicBot](https://github.com/AshokShau/TgMusicBot)  
🔹**Description**: A powerful, async Telegram music bot that streams high-quality audio in group voice chats using **PyTdBot** and **PyTgCalls**. For Pyrogram users, check out the [Pyro Branch](https://github.com/AshokShau/TgMusicBot/tree/pyro). The bot supports **multiple platforms**, including YouTube, Spotify, Apple Music, SoundCloud, JioSaavn, and more. Optimized with Docker for seamless deployment. 

🔹**Repository Link**: [AshokShau/TgMusicBot](https://github.com/AshokShau/TgMusicBot)  

🔹**Try It Out**: [@FallenBeatzBot](https://t.me/FallenBeatzBot)

📈 Why TgMusicBot?  
- Built with the latest [PyTgCalls ](https://pypi.org/project/py-tgcalls) 
- Fully async for optimal performance  
- Supports multi-client usage  
- Public repo with clean, modular code  

Looking forward to the contest! 🔥

**Reply by AdityaxDz** [reply]
Best Telegram Music bot project ever 🫡

**Reply by NishulDhakar** [reply]
Amazing work.

**Reply by MRJK4781** [reply]
Amazing free music bot project till date .

**Reply by AnonymousX1025** [reply]
Crazy 👄

**Reply by Ak4ssh** [reply]
Amazing work bro 😩

**Reply by Xytra-zn** [reply]
Best ever open source music bot project 🙌

**Reply by damian-bots** [reply]
This is best Music Bot Source I have ever seen ❣️

**Reply by tusarhushena** [reply]
Amazing work 🤖

**Reply by music-miko** [reply]
superb performance

**Reply by tusarhushena** [reply]
Amazing Codes 🔥🔥

**Reply by Laky-64** [reply]
Your repository was choosed, thank you for the participation

**Comment by nothing-bots** [comment]
You're amazing bro 🤧

**Comment by about-tosu** [comment]
Superb 🔥🤧
